front/src/sections/JumbotronSection/JumbotronSection.module.scss

```diff
@@ -0,0 +1,15 @@
+.jumbotron_section {
+
+}
+
+.jumbotron_section_inner {
+
+}
+
+.content_wrapper {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
```

Comment:
Jeżeli chcemy, żeby zawsze był taki sam rozmiar naszych sekcji to używamy **_repeat(2, minmax(0, 1fr))_**